Runtime/ui/renderer/common/draw_cmd.cs

```diff
@@ -405,7 +405,6 @@ public class uiDrawTextBlob : uiDrawCmd {
         }
 
         public override void clear() {
-            this.textBlob = null;
```

Comment:
why not release textBlob ? Maybe it is because you have made textBlob struct, so it cannot be null. If this is the reason, is it better to make the property to "TextBlob?" ?